mysql-test/suite/galera/t/MDEV-32738.test

```diff
@@ -0,0 +1,73 @@
+#
+# MDEV-32738: Don't BF-abort high-priority transaction waiting on a lock in InnoDB.
+#
+# DML transactions on FK-child tables also get table locks on FK-parent tables.
+# If there is a DML transaction holding such a lock, and a TOI transaction arrives,
+# the latter BF-aborts the former and puts itself into a waiting state.
+# If at this moment another DML transaction on FK-child table arrives, it doesn't
```

Comment:
Please elaborate what "DML transaction on FK-child table arrives" means. Is this a local or remote transaction?